dist/stat_data.py

```diff
@@ -384,6 +384,7 @@ def __init__(self, name, desc, flags=''):
     ##########################################
     # Checkpoint statistics
     ##########################################
+    CheckpointStat('checkpoint_cleanup_success', 'checkpoint cleanup successful calls'),
```

Comment:
Should we track the failures too?

Reply:
A failure is a fatal error similar to eviction threads